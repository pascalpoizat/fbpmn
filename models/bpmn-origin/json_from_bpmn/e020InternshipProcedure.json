{
    "isInterrupt": {},
    "catN": {
        "ReceiveTask_146lfh4": "ReceiveTask",
        "Task_0z00tzq": "SendTask",
        "Task_05rcgao": "AbstractTask",
        "EventBasedGateway_1d92ix7": "EventBasedGateway",
        "ExclusiveGateway_1pvytgl": "XorGateway",
        "Task_12w7z0s": "AbstractTask",
        "Task_10ik9d4": "SendTask",
        "SendTask_1sfy3aq": "AbstractTask",
        "InternshipDelegate_": "Process",
        "IntermediateCatchEvent_0oxyhte": "CatchMessageIntermediateEvent",
        "StartEvent_1wgj96l": "NoneStartEvent",
        "Task_06e9es5": "AbstractTask",
        "Student_": "Process",
        "Task_0k3desy": "SendTask",
        "EndEvent_0zeqiic": "NoneEndEvent",
        "Task_0ieueaf": "SendTask",
        "Task_1awa0vf": "AbstractTask",
        "InternshipOffice_": "Process",
        "EndEvent_131pwct": "NoneEndEvent",
        "Task_16mub7w": "AbstractTask",
        "Task_0zi619x": "ReceiveTask",
        "ExclusiveGateway_1wor220": "XorGateway",
        "EndEvent_0g8hfyq": "NoneEndEvent",
        "StartEvent_0sg8ueb": "MessageStartEvent",
        "SendTask_0bu2egl": "SendTask",
        "Task_1ukgtgd": "SendTask",
        "ExclusiveGateway_10r796g": "XorGateway",
        "Task_0kueqty": "SendTask",
        "Company_": "Process",
        "IntermediateCatchEvent_180guhd": "CatchMessageIntermediateEvent",
        "StartEvent_0efqept": "NoneStartEvent",
        "Task_0y40xht": "ReceiveTask",
        "ExclusiveGateway_1ojvmkc": "XorGateway",
        "StartEvent_0y07koo": "MessageStartEvent",
        "Task_13r7bmt": "SendTask",
        "Task_16a1dy1": "AbstractTask",
        "ReceiveTask_1eiaojy": "ReceiveTask",
        "EndEvent_0r34c4w": "NoneEndEvent",
        "ExclusiveGateway_025el3u": "XorGateway"
    },
    "catE": {
        "SequenceFlow_1716tm6": "NormalSequenceFlow",
        "SequenceFlow_0ye9pwd": "NormalSequenceFlow",
        "SequenceFlow_0cxrm8j": "NormalSequenceFlow",
        "SequenceFlow_1ssp09f": "NormalSequenceFlow",
        "MessageFlow_0whwr08": "MessageFlow",
        "MessageFlow_0n7crx1": "MessageFlow",
        "SequenceFlow_19jde9o": "NormalSequenceFlow",
        "SequenceFlow_0oaajbr": "NormalSequenceFlow",
        "SequenceFlow_0bferc8": "NormalSequenceFlow",
        "SequenceFlow_0upqxnz": "NormalSequenceFlow",
        "MessageFlow_1exti3v": "MessageFlow",
        "SequenceFlow_1hv2qod": "NormalSequenceFlow",
        "MessageFlow_0qzpag3": "MessageFlow",
        "SequenceFlow_19fg7gc": "NormalSequenceFlow",
        "SequenceFlow_1nxvmw1": "NormalSequenceFlow",
        "SequenceFlow_1nsmz5l": "NormalSequenceFlow",
        "SequenceFlow_1e21rh4": "NormalSequenceFlow",
        "SequenceFlow_0epxxie": "NormalSequenceFlow",
        "SequenceFlow_0f1t7q4": "NormalSequenceFlow",
        "MessageFlow_0n79ikw": "MessageFlow",
        "SequenceFlow_0lkr4ww": "NormalSequenceFlow",
        "SequenceFlow_07e105b": "NormalSequenceFlow",
        "MessageFlow_0vajuh0": "MessageFlow",
        "SequenceFlow_12e7qby": "NormalSequenceFlow",
        "SequenceFlow_1wxr7td": "NormalSequenceFlow",
        "SequenceFlow_0ol0c8s": "NormalSequenceFlow",
        "SequenceFlow_07c1y6g": "NormalSequenceFlow",
        "SequenceFlow_04fa4c4": "NormalSequenceFlow",
        "SequenceFlow_0tbrrh9": "NormalSequenceFlow",
        "SequenceFlow_0woxsim": "NormalSequenceFlow",
        "SequenceFlow_0nlul5d": "NormalSequenceFlow",
        "MessageFlow_1r2202w": "MessageFlow",
        "SequenceFlow_1a0w8i4": "NormalSequenceFlow",
        "SequenceFlow_09fa10u": "NormalSequenceFlow",
        "SequenceFlow_1v26cki": "NormalSequenceFlow",
        "SequenceFlow_1ifwhto": "NormalSequenceFlow",
        "MessageFlow_1uu4df2": "MessageFlow",
        "SequenceFlow_0rqes4b": "NormalSequenceFlow",
        "SequenceFlow_1hrqezf": "NormalSequenceFlow",
        "SequenceFlow_1h50tfu": "NormalSequenceFlow",
        "SequenceFlow_007i0gb": "NormalSequenceFlow",
        "SequenceFlow_036yd4w": "NormalSequenceFlow"
    },
    "containN": {
        "InternshipDelegate_": [
            "StartEvent_0sg8ueb",
            "SendTask_0bu2egl",
            "ReceiveTask_1eiaojy",
            "EndEvent_0r34c4w",
            "SendTask_1sfy3aq"
        ],
        "Student_": [
            "StartEvent_0efqept",
            "ExclusiveGateway_1pvytgl",
            "ExclusiveGateway_10r796g",
            "Task_05rcgao",
            "Task_12w7z0s",
            "Task_0kueqty",
            "Task_16mub7w",
            "Task_0y40xht",
            "EndEvent_131pwct"
        ],
        "InternshipOffice_": [
            "IntermediateCatchEvent_180guhd",
            "IntermediateCatchEvent_0oxyhte",
            "ExclusiveGateway_025el3u",
            "ReceiveTask_146lfh4",
            "EndEvent_0zeqiic",
            "StartEvent_0y07koo",
            "EventBasedGateway_1d92ix7"
        ],
        "Company_": [
            "Task_1ukgtgd",
            "Task_0k3desy",
            "EndEvent_0g8hfyq",
            "Task_0z00tzq",
            "Task_1awa0vf",
            "Task_13r7bmt",
            "Task_10ik9d4",
            "Task_06e9es5",
            "Task_0zi619x",
            "ExclusiveGateway_1ojvmkc",
            "Task_16a1dy1",
            "Task_0ieueaf",
            "StartEvent_1wgj96l",
            "ExclusiveGateway_1wor220"
        ]
    },
    "messageE": {
        "MessageFlow_0whwr08": "documentation",
        "MessageFlow_0n7crx1": "access",
        "MessageFlow_1exti3v": "internship completion",
        "MessageFlow_0qzpag3": "registration",
        "MessageFlow_0n79ikw": "proposal",
        "MessageFlow_0vajuh0": "termination notification",
        "MessageFlow_1r2202w": "agreement",
        "MessageFlow_1uu4df2": "login"
    },
    "timeInformation": {},
    "targetE": {
        "SequenceFlow_1716tm6": "Task_0kueqty",
        "SequenceFlow_0ye9pwd": "Task_06e9es5",
        "SequenceFlow_0cxrm8j": "ExclusiveGateway_025el3u",
        "SequenceFlow_1ssp09f": "EventBasedGateway_1d92ix7",
        "MessageFlow_0whwr08": "StartEvent_0sg8ueb",
        "MessageFlow_0n7crx1": "StartEvent_0y07koo",
        "SequenceFlow_19jde9o": "EndEvent_131pwct",
        "SequenceFlow_0oaajbr": "Task_0z00tzq",
        "SequenceFlow_0bferc8": "Task_16a1dy1",
        "SequenceFlow_0upqxnz": "ExclusiveGateway_1ojvmkc",
        "MessageFlow_1exti3v": "Task_0y40xht",
        "SequenceFlow_1hv2qod": "Task_05rcgao",
        "MessageFlow_0qzpag3": "IntermediateCatchEvent_0oxyhte",
        "SequenceFlow_19fg7gc": "Task_13r7bmt",
        "SequenceFlow_1nxvmw1": "ReceiveTask_146lfh4",
        "SequenceFlow_1nsmz5l": "Task_16mub7w",
        "SequenceFlow_1e21rh4": "Task_12w7z0s",
        "SequenceFlow_0epxxie": "ExclusiveGateway_10r796g",
        "SequenceFlow_0f1t7q4": "EndEvent_0zeqiic",
        "MessageFlow_0n79ikw": "ReceiveTask_146lfh4",
        "SequenceFlow_0lkr4ww": "IntermediateCatchEvent_0oxyhte",
        "SequenceFlow_07e105b": "Task_0y40xht",
        "MessageFlow_0vajuh0": "ReceiveTask_1eiaojy",
        "SequenceFlow_12e7qby": "Task_0ieueaf",
        "SequenceFlow_1wxr7td": "Task_1ukgtgd",
        "SequenceFlow_0ol0c8s": "EndEvent_0r34c4w",
        "SequenceFlow_07c1y6g": "SendTask_1sfy3aq",
        "SequenceFlow_04fa4c4": "ExclusiveGateway_1pvytgl",
        "SequenceFlow_0tbrrh9": "Task_10ik9d4",
        "SequenceFlow_0woxsim": "ReceiveTask_1eiaojy",
        "SequenceFlow_0nlul5d": "EndEvent_0g8hfyq",
        "MessageFlow_1r2202w": "Task_0zi619x",
        "SequenceFlow_1a0w8i4": "ExclusiveGateway_10r796g",
        "SequenceFlow_09fa10u": "IntermediateCatchEvent_180guhd",
        "SequenceFlow_1v26cki": "Task_1awa0vf",
        "SequenceFlow_1ifwhto": "SendTask_0bu2egl",
        "MessageFlow_1uu4df2": "IntermediateCatchEvent_180guhd",
        "SequenceFlow_0rqes4b": "ExclusiveGateway_1ojvmkc",
        "SequenceFlow_1hrqezf": "Task_0k3desy",
        "SequenceFlow_1h50tfu": "ExclusiveGateway_025el3u",
        "SequenceFlow_007i0gb": "ExclusiveGateway_1wor220",
        "SequenceFlow_036yd4w": "Task_0zi619x"
    },
    "name": "e020InternshipProcedure",
    "attached": {},
    "containE": {
        "InternshipDelegate_": [
            "SequenceFlow_07c1y6g",
            "SequenceFlow_0woxsim",
            "SequenceFlow_1ifwhto",
            "SequenceFlow_0ol0c8s"
        ],
        "Student_": [
            "SequenceFlow_19jde9o",
            "SequenceFlow_07e105b",
            "SequenceFlow_1nsmz5l",
            "SequenceFlow_1a0w8i4",
            "SequenceFlow_1hv2qod",
            "SequenceFlow_04fa4c4",
            "SequenceFlow_0epxxie",
            "SequenceFlow_1716tm6",
            "SequenceFlow_1e21rh4"
        ],
        "InternshipOffice_": [
            "SequenceFlow_09fa10u",
            "SequenceFlow_0cxrm8j",
            "SequenceFlow_0lkr4ww",
            "SequenceFlow_1h50tfu",
            "SequenceFlow_1nxvmw1",
            "SequenceFlow_0f1t7q4",
            "SequenceFlow_1ssp09f"
        ],
        "Company_": [
            "SequenceFlow_0nlul5d",
            "SequenceFlow_0oaajbr",
            "SequenceFlow_1v26cki",
            "SequenceFlow_0bferc8",
            "SequenceFlow_0rqes4b",
            "SequenceFlow_19fg7gc",
            "SequenceFlow_0upqxnz",
            "SequenceFlow_0tbrrh9",
            "SequenceFlow_0ye9pwd",
            "SequenceFlow_007i0gb",
            "SequenceFlow_036yd4w",
            "SequenceFlow_1wxr7td",
            "SequenceFlow_1hrqezf",
            "SequenceFlow_12e7qby"
        ]
    },
    "edges": [
        "MessageFlow_1r2202w",
        "MessageFlow_0vajuh0",
        "MessageFlow_1exti3v",
        "MessageFlow_0whwr08",
        "MessageFlow_1uu4df2",
        "MessageFlow_0qzpag3",
        "MessageFlow_0n79ikw",
        "MessageFlow_0n7crx1",
        "SequenceFlow_19jde9o",
        "SequenceFlow_07e105b",
        "SequenceFlow_1nsmz5l",
        "SequenceFlow_1a0w8i4",
        "SequenceFlow_1hv2qod",
        "SequenceFlow_04fa4c4",
        "SequenceFlow_0epxxie",
        "SequenceFlow_1716tm6",
        "SequenceFlow_1e21rh4",
        "SequenceFlow_0nlul5d",
        "SequenceFlow_0oaajbr",
        "SequenceFlow_1v26cki",
        "SequenceFlow_0bferc8",
        "SequenceFlow_0rqes4b",
        "SequenceFlow_19fg7gc",
        "SequenceFlow_0upqxnz",
        "SequenceFlow_0tbrrh9",
        "SequenceFlow_0ye9pwd",
        "SequenceFlow_007i0gb",
        "SequenceFlow_036yd4w",
        "SequenceFlow_1wxr7td",
        "SequenceFlow_1hrqezf",
        "SequenceFlow_12e7qby",
        "SequenceFlow_09fa10u",
        "SequenceFlow_0cxrm8j",
        "SequenceFlow_0lkr4ww",
        "SequenceFlow_1h50tfu",
        "SequenceFlow_1nxvmw1",
        "SequenceFlow_0f1t7q4",
        "SequenceFlow_1ssp09f",
        "SequenceFlow_07c1y6g",
        "SequenceFlow_0woxsim",
        "SequenceFlow_1ifwhto",
        "SequenceFlow_0ol0c8s"
    ],
    "messages": [
        "agreement",
        "termination notification",
        "internship completion",
        "documentation",
        "login",
        "registration",
        "proposal",
        "access"
    ],
    "sourceE": {
        "SequenceFlow_1716tm6": "Task_16mub7w",
        "SequenceFlow_0ye9pwd": "Task_0zi619x",
        "SequenceFlow_0cxrm8j": "IntermediateCatchEvent_180guhd",
        "SequenceFlow_1ssp09f": "StartEvent_0y07koo",
        "MessageFlow_0whwr08": "Task_10ik9d4",
        "MessageFlow_0n7crx1": "Task_0ieueaf",
        "SequenceFlow_19jde9o": "Task_0y40xht",
        "SequenceFlow_0oaajbr": "Task_1awa0vf",
        "SequenceFlow_0bferc8": "Task_10ik9d4",
        "SequenceFlow_0upqxnz": "Task_1ukgtgd",
        "MessageFlow_1exti3v": "SendTask_0bu2egl",
        "SequenceFlow_1hv2qod": "ExclusiveGateway_10r796g",
        "MessageFlow_0qzpag3": "Task_1ukgtgd",
        "SequenceFlow_19fg7gc": "ExclusiveGateway_1ojvmkc",
        "SequenceFlow_1nxvmw1": "ExclusiveGateway_025el3u",
        "SequenceFlow_1nsmz5l": "ExclusiveGateway_1pvytgl",
        "SequenceFlow_1e21rh4": "Task_0kueqty",
        "SequenceFlow_0epxxie": "StartEvent_0efqept",
        "SequenceFlow_0f1t7q4": "ReceiveTask_146lfh4",
        "MessageFlow_0n79ikw": "Task_13r7bmt",
        "SequenceFlow_0lkr4ww": "EventBasedGateway_1d92ix7",
        "SequenceFlow_07e105b": "Task_12w7z0s",
        "MessageFlow_0vajuh0": "Task_0z00tzq",
        "SequenceFlow_12e7qby": "StartEvent_1wgj96l",
        "SequenceFlow_1wxr7td": "ExclusiveGateway_1wor220",
        "SequenceFlow_0ol0c8s": "SendTask_0bu2egl",
        "SequenceFlow_07c1y6g": "StartEvent_0sg8ueb",
        "SequenceFlow_04fa4c4": "Task_05rcgao",
        "SequenceFlow_0tbrrh9": "Task_06e9es5",
        "SequenceFlow_0woxsim": "SendTask_1sfy3aq",
        "SequenceFlow_0nlul5d": "Task_0z00tzq",
        "MessageFlow_1r2202w": "Task_0kueqty",
        "SequenceFlow_1a0w8i4": "ExclusiveGateway_1pvytgl",
        "SequenceFlow_09fa10u": "EventBasedGateway_1d92ix7",
        "SequenceFlow_1v26cki": "Task_16a1dy1",
        "SequenceFlow_1ifwhto": "ReceiveTask_1eiaojy",
        "MessageFlow_1uu4df2": "Task_0k3desy",
        "SequenceFlow_0rqes4b": "Task_0k3desy",
        "SequenceFlow_1hrqezf": "ExclusiveGateway_1wor220",
        "SequenceFlow_1h50tfu": "IntermediateCatchEvent_0oxyhte",
        "SequenceFlow_007i0gb": "Task_0ieueaf",
        "SequenceFlow_036yd4w": "Task_13r7bmt"
    },
    "nodes": [
        "Student_",
        "Company_",
        "InternshipOffice_",
        "InternshipDelegate_",
        "StartEvent_0efqept",
        "ExclusiveGateway_1pvytgl",
        "ExclusiveGateway_10r796g",
        "Task_05rcgao",
        "Task_12w7z0s",
        "Task_0kueqty",
        "Task_16mub7w",
        "Task_0y40xht",
        "EndEvent_131pwct",
        "Task_1ukgtgd",
        "Task_0k3desy",
        "EndEvent_0g8hfyq",
        "Task_0z00tzq",
        "Task_1awa0vf",
        "Task_13r7bmt",
        "Task_10ik9d4",
        "Task_06e9es5",
        "Task_0zi619x",
        "ExclusiveGateway_1ojvmkc",
        "Task_16a1dy1",
        "Task_0ieueaf",
        "StartEvent_1wgj96l",
        "ExclusiveGateway_1wor220",
        "IntermediateCatchEvent_180guhd",
        "IntermediateCatchEvent_0oxyhte",
        "ExclusiveGateway_025el3u",
        "ReceiveTask_146lfh4",
        "EndEvent_0zeqiic",
        "StartEvent_0y07koo",
        "EventBasedGateway_1d92ix7",
        "StartEvent_0sg8ueb",
        "SendTask_0bu2egl",
        "ReceiveTask_1eiaojy",
        "EndEvent_0r34c4w",
        "SendTask_1sfy3aq"
    ],
    "nameN": {
        "ReceiveTask_146lfh4": "Upload Internship Proposal",
        "Task_0z00tzq": "Notify Delegate",
        "Task_05rcgao": "Check ECTS Amount",
        "ExclusiveGateway_1pvytgl": "Are ECTS sufficient?",
        "Task_12w7z0s": "Do Internship",
        "Task_10ik9d4": "Send Internship Documentation",
        "SendTask_1sfy3aq": "Assign Tutor",
        "IntermediateCatchEvent_0oxyhte": "Registration",
        "StartEvent_1wgj96l": "Propose Internship",
        "Task_06e9es5": "Accept Internship",
        "Task_0k3desy": "Log In to Internship Program",
        "Task_0ieueaf": "Access Unicam Internship Program",
        "Task_1awa0vf": "Terminate Internship",
        "EndEvent_131pwct": "Internship Performed",
        "Task_16mub7w": "Attend Occupational Safety Course",
        "Task_0zi619x": "Internship Agreement",
        "ExclusiveGateway_1wor220": "Ã Does the company have a convention?",
        "EndEvent_0g8hfyq": "Internship Finished",
        "StartEvent_0sg8ueb": "Register Internship",
        "SendTask_0bu2egl": "Register Internship Completion",
        "Task_1ukgtgd": "Register to Internship Program",
        "Task_0kueqty": "Apply an Available Internship",
        "IntermediateCatchEvent_180guhd": "Login",
        "StartEvent_0efqept": "Internship\nApplication",
        "Task_0y40xht": "Internship Registration",
        "StartEvent_0y07koo": "Manage \nCompany",
        "Task_13r7bmt": "Insert Internship Announcement",
        "Task_16a1dy1": "Internship",
        "ReceiveTask_1eiaojy": "Internship Completion Notification",
        "EndEvent_0r34c4w": "Internship Registered"
    }
}