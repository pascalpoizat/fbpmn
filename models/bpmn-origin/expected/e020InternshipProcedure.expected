Working in /tmp/e020InternshipProcedure.ipSpW with 1 worker(s)
[92mtransformation done[0m
<<"Processes=", 4, "Nodes=", 39, "Gateway=", 6, "SF=", 34, "MF=", 8>>
---------- Network01Bag ----------
[X] Prop01Type
     states=3558 trans=11035 depth=52
[X] Prop02Safe
     states=3558 trans=11035 depth=52
[X] Prop03Sound
     states=3558 trans=11035 depth=52
[X] Prop04MsgSound
     states=3558 trans=11035 depth=52
---------- Network02FifoPair ----------
[X] Prop01Type
     states=3558 trans=11035 depth=52
[X] Prop02Safe
     states=3558 trans=11035 depth=52
[X] Prop03Sound
     states=3558 trans=11035 depth=52
[X] Prop04MsgSound
     states=3558 trans=11035 depth=52
---------- Network03Causal ----------
[X] Prop01Type
     states=3558 trans=11035 depth=52
[X] Prop02Safe
     states=3558 trans=11035 depth=52
[X] Prop03Sound
     states=3558 trans=11035 depth=52
[X] Prop04MsgSound
     states=3558 trans=11035 depth=52
---------- Network04Inbox ----------
[X] Prop01Type
     states=3558 trans=11035 depth=52
[X] Prop02Safe
     states=3558 trans=11035 depth=52
[X] Prop03Sound
     states=3558 trans=11035 depth=52
[X] Prop04MsgSound
     states=3558 trans=11035 depth=52
---------- Network05Outbox ----------
[X] Prop01Type
     states=1878 trans=5115 depth=52
[X] Prop02Safe
     states=1878 trans=5115 depth=52
[X] Prop03Sound
     states=1878 trans=5115 depth=52
[X] Prop04MsgSound
     states=1878 trans=5115 depth=52
---------- Network06Fifo ----------
[X] Prop01Type
     states=2138 trans=5654 depth=52
[X] Prop02Safe
     states=2138 trans=5654 depth=52
[X] Prop03Sound
     states=2138 trans=5654 depth=52
[X] Prop04MsgSound
     states=2138 trans=5654 depth=52
---------- Network07RSC ----------
[X] Prop01Type
     states=1030 trans=2695 depth=52
[X] Prop02Safe
     states=1030 trans=2695 depth=52
[ ] Prop03Sound
     states=1030 trans=2695 depth=
[ ] Prop04MsgSound
     states=1030 trans=2695 depth=
done.
